### ISSUE TITLE
Unable create webserver, such as socket bind\listen\accept

### ISSUE THREAD

**Issue by ChahidTALHA** [sipeed/MaixPy-v1#315]
Hi everyone,
I would like to create a webserver in my Maixduino board, and i see that it is impossible to implement those micropython functions ( bind .. listen.. accept). I have found next in the source code of the firmware that all those methods are commented out.
Is there any way to implement these methods in order to proceed more on my project. 
Thanks in advance.

**Comment by junhuanchen** [comment]
You are right, these methods conflict on different network card implementations.

You can wait to achieve.

**Comment by JohnsonLuo-2019** [comment]
MaixPy uses SPI to communicate with ESP32. K210 is SPI master, ESP32 is SPI slave. There is no way for slave ESP32 to interrupt master K210. This is why MaixPy does not implement bind/listen/... server functionality.
It is possible to create web server if the communication between K210 and ESP32 is UART.
